assets/globals.d.ts

```diff
@@ -8,10 +8,7 @@ interface Window {
     };
 
     newsroom: {
-        client_config: {
-            debug: boolean;
-            news_only_filter?: any;
-        }
+        client_config: any;
```

Comment:
@tomaskikutis I added `IClientConfig` to `newshub-api.d.ts` but could not re-use it here. So for now I made this `any` and we can use `IClientConfig` in our code